packages/pancake-sass/src/sass.js

```diff
@@ -44,7 +44,10 @@ export const GetPath = ( module, modules, baseLocation, npmOrg ) => {
 	const npmOrgs = npmOrg.split( ' ' );
 	let location;
 	npmOrgs.forEach( org => {
-		if( baseLocation.includes( org ) ){
+		if( baseLocation.includes( org ) && process.platform === 'win32' ){
+			location = baseLocation.replace( `${ org }\\`, '' );
+		}
+		else if( baseLocation.includes( org ) ){
```

Comment:
I would recommend to use `path.posix` here. Linux subsystem supports both slashes etc...

Reply:
@dominikwilkowski Yeah good point, path now resolves with `path.posix.resolve()`